File: microservices/usuarios-service/src/main/java/cl/duoc/levelup/usuarios_service/dto/UsuarioDTO.java
package cl.duoc.levelup.usuarios_service.dto;

import lombok.Data;

@Data
public class UsuarioDTO {

    private Long id;
    private String nombre;
    private String email;
    private Integer edad;
    private String preferencias;
    private String direccion;
    private String rol;
    private String password;

    public UsuarioDTO() {}
}

File: mobile-app/app/src/main/java/com/levelup/data/remote/dto/UsuarioDto.kt
package com.levelup.data.remote.dto

data class UsuarioDto(
    val id: Long?,
    val nombre: String,
    val email: String,
    val edad: Int?,
    val preferencias: String?,
    val direccion: String?,
    val rol: String?,
    val password: String
)
